infrastructure/terraform/staging/browser/main.tf

```
@@ -3,4 +3,6 @@ module "browser" {
   bucket_name = "insight-${module.project_vars.environment}-static"
   environment = module.project_vars.environment
   region      = module.project_vars.region
+  domain = "snuderls.eu"
```

This should probable be:

```suggestion
  alias = "static.devtest.snuderls.eu"
```

---

we need base domain too, to link to cloudfront zone. or we could just enter zone_id directly, which is maybe better since TF won't try to manage it.